MonitoringTools/usr/local/bin/slottifier.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python
+import json
+import sys
+import os
+import requests
+from time import gmtime, strftime
+from subprocess import Popen, PIPE
+from ConfigParser import SafeConfigParser
+import math
+from time import sleep
+
+
+# Read from config file
+parser = SafeConfigParser()
+try:
+   parser.read('/etc/influxdb.conf')
+   host = parser.get('db', 'host')
+   database = parser.get('db', 'database')
+   username = parser.get('auth', 'username')
+   password = parser.get('auth', 'password')
+   instance = parser.get('cloud','instance')
+except:
+   print 'Unable to read from config file'
+   sys.exit(1)
+
+url = 'http://'+host+'/write?db='+database +'&precision=s'
```

Comment:
We have these LOC 4 times across multiple files, IMO this is worth moving into another file and have the other scripts import it so we only have to change it once